src/test/java/hudson/maven/CaseResultTest.java

```diff
@@ -44,8 +45,8 @@
      */
     @Bug(4257)
     @Test public void mavenErrorMsgAndStacktraceRender() throws Exception {
-        j.configureDefaultMaven();
-        MavenModuleSet m = j.createMavenProject("maven-render-test");
+        ToolInstallations.configureMaven3();
```

Comment:
The FindBugs plugin in this sample did not run on Java 8, so I had to update the plugin, but then the new version did not run on Maven 2, so I had to update that too.